docs/source/scientific_reference/1_time_series.rst

```diff
@@ -0,0 +1,165 @@
+.. _sciref_series:
+
+Scientific reference: time series
+=================================
+
+``sktime``'s conceptual model is based on a number of "time series" related mathematical
+objects:
+
+* "time series", observations of a single process made over time.
+Example: average temperature in London, on a given day.
+* "time series panels", collections of multiple time series, independently observed.
+Example: fever curves of 10 specific hospital patients, for the time in the hospital.
+* "hierarchical time series", collections of multiple time series where individual
+instances are observed at instances defined by variables defining the hierarchy.
+Example: gummy bear sales at 100 specific stores, in 10 federal states.
+
+This document provides a formal reference for the above.
+The objects are defined in terms of:
+
+* abstract data types (ADT) - defines domain, type and common operations
+* scientific types (scitype) - ADT plus statistical assumptions
+
+
+Primitive values - the `Primitives` scitype
+-------------------------------------------
+
+The following sets/domains are called basic primitive domains:
+
+* the real numbers, i.e., the set :math:`\mathbb{R}`
+* the set of boolean truth value, i.e., the two-element set containing `True` or `False`
+* the set of all finite length strings of ASCII characters
+* any other finite set :math:`\mathcal{C}`, called "categoriy set" in this context
+
+A set/domain is called primitive domain if it is a basic primitive domain, or
```

Comment:
:mage: cannot say that I'm able to follow. Is this user facing documentation or internal dev documentation?

Reply:
user facing, for users who are familiar with math style formalism.
This is a formal ADT definition, so will involve mathematical definitions.

Reply:
a great introduction into this kind of language is "Daepp U, Gorkin P (2011) - Reading, Writing, and Proving (2nd ed)"

Reply:
the specific lines, btw, define the domain of a data frame row

Reply:
the doc is meant to complement the API reference

Reply:
I'm only saying that it reads unnecessarily formal and opaque, maybe saying "row of a data frame" is ok. I don't immediately see any value in introducing this new terminology.